sandbox/modules/identity/src/main/java/org/opensearch/identity/rest/user/resetpassword/ResetPasswordAction.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.rest.user.resetpassword;
+
+import org.opensearch.action.ActionType;
+
+/**
+ * Action type for creating or updating a user
+ */
+public class ResetPasswordAction extends ActionType<ResetPasswordResponse> {
+
+    public static final ResetPasswordAction INSTANCE = new ResetPasswordAction();
+
+    // TODO : revisit this action type
+    public static final String NAME = "cluster:admin/user/resetpassword";
```

Comment:
Since this API is used by all users, it may need a different namespace than `cluster:admin/user/*` which are a group of actions intended for the cluster admin. When it comes to authorizing this request, all users of the cluster should be able to call and use this API without being granted permission for it.